styles/styles.css

```diff
@@ -1584,10 +1632,11 @@ body {
   margin-bottom: 32px;
   font-family: "PT Sans", sans-serif;
```

Comment:
```suggestion
  font-family: inherit;
```
вообще везде где можно)